osf/models/user.py

```diff
@@ -64,6 +65,30 @@ def get_default_mailing_lists():
     ),
 }
 
+def get_default_mailing_lists():
+    return {'Open Science Framework Help': True}
+
+
+class Email(ObjectIDMixin, BaseModel):
+    email = models.EmailField(validators=[validate_email])
```

Comment:
Yep, but our email validator also checks against blacklisted domains (for spam).

https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/validators.py#L89